.changeset/short-dancers-follow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/api-generator": patch
+---
+
+Build with tsdown instead of tsc
```

Comment:
We could probably omit the changeset since this shouldn't change how the package behaves. But I'd leave it in for information purposes und to easier track down problems related to this change if any should arise.